src/app/app.component.html

```diff
@@ -1,5 +1,2 @@
-<app-header *ngIf="showHead"></app-header>
-<!-- <app-carddesign> </app-carddesign>
-<app-questions></app-questions> -->
+<app-header></app-header>
```

Comment:
Can we make route of athorization inside full layout

Reply:
Using route for authorization inside full layout is not necessary and using separately is a good option